test/services/testsvc.jl

```diff
@@ -22,42 +23,42 @@ end
 close(channel::TestRpcChannel) = close(channel.sock)
 
 type SvcHeader
-    method::ASCIIString
+    method::String
```

Comment:
if you still support 0.4 you'll want this to be Compat.String or it'll throw a depwarn